README.md

```diff
@@ -0,0 +1,40 @@
+# How to add a script
+1. Select the right category
+  * Development
+  * Functions
+  * Games
+  * Graphics
+  * Internet
+  
+2. Create a new folder for your script inside that category. The folder structure should be:
+```
+category
+└── script-name
```

Comment:
The structure is actually category -> application - script.
Here "Online" is the name of the script